cmd/krel/cmd/release_notes.go

```diff
@@ -66,7 +67,10 @@ permissions to your fork of k/sig-release and k-sigs/release-notes.`,
 }
 
 type releaseNotesOptions struct {
-	tag string
+	tag            string
+	sigReleaseFork string
```

Comment:
Let's break this up into two fields, so there's no need for string manipulation later.

Use something like:
```go
draftOrg    string
draftRepo   string
```

Reply:
> draftOrg

It might be a user and not an organisation, e.g., `JamesLaverack/sig-release`. Is there a generic term, in GitHub's API, for a "user or organisation"? When dealing with the API do we have to know the difference?

Reply:
@JamesLaverack -- I think `org` is the correct term from the API perspective.
A user of this tool should understand `--draft-org` to mean "identifier of some user or org", so we should make that clear in the flag's note.

Reply:
Ok, the new flags are done in the new revision. The flag uses `org` but in git.go I used `owner` because other parts of the code use that.